client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -202,105 +202,98 @@ const Sidebar = ( {
 	return (
 		<>
 			{ site && <QueryMembershipsSettings siteId={ site.ID } source="launchpad" /> }
-			<div className="launchpad__sidebar">
```

Comment:
The changes in this file only remove these 2 wrapper divs because it was moved to the `LaunchpadContainer` component.